### DISCUSSION TITLE
why navigateTo in middleware always redirect to 301, event i set redirect Code to 302?

### DISCUSSION THREAD

**Discussion by tsulatsitamim** [nuxt/nuxt#16823] [Questions]
```
// /middleware/auth.ts

if (to.path === '/login') {
    return token.value ? navigateTo('/', { redirectCode: 302 }) : undefined
}
```

<img width="360" alt="Screen Shot 2022-09-21 at 11 26 34" src="https://user-images.githubusercontent.com/5946791/191414290-9e6fcf76-0fab-4b2e-adda-448719ee4cd4.png">

**Comment by TheAlexLichter** [comment]
Cannot reproduce this via https://stackblitz.com/edit/https-github-comnuxtframeworkdiscussions7706?file=README.md,pages%2Findex.vue,pages%2Ftest.vue

<img width="897" alt="image" src="https://user-images.githubusercontent.com/640208/191735735-feb84280-dd69-4081-a57e-ff759c15ea77.png">
